plugins/branch/branch.plugin.zsh

```diff
@@ -0,0 +1,26 @@
+# Branch: displays the current Git or Mercurial branch fast.
+# Victor Torres <vpaivatorres@gmail.com>
+# Oct 2, 2015
+
+function branch_prompt_info() {
+  # Defines path as current directory
+  path=$(pwd)
```

Comment:
The `path` variable should be local, otherwise it leaks out and overrides the `$path` variable which is the env variable in zsh. Also, use special variables whenever you can instead of running a command: `$PWD` is a variable that already contains the cwd.